index.html

```diff
@@ -12,13 +12,24 @@
 </head>
 
 <body>
-    <div id="app">
+<div id="app">
         <about></about>
-        <navbar v-bind:suites="suites" v-if="loaded"></navbar>
-        <template v-for="(suite,index) in suites">
-            <task-suite v-bind:suite="suite" v-bind:index="index"></task-suite>
-        </template>
-    </div>
+        <div class="r_flex_container">
```

Comment:
what does `r flex container` means? if possible, add more intuitive names to css classes

Reply:
You're right. flex container was because the property flex that allow me to fix the space between lasts visible tasks and bottom bar, and r was only  to differenciate it. 
I'll change names to made them more intuitive